src/js/admin/index.js

```diff
@@ -0,0 +1,20 @@
+document.querySelector( '.media-single-image button.browse' ).addEventListener( 'click', selectImage );
```

Comment:
I think we should reinstate the built admin page js file that was removed in #518 to keep consistency with the rest of the scripts and provide the expectation of a common runtime when working in JS.

That would make it fairly straightforward to use the [MediaUpload](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-media-utils/#mediaupload) component ([usage](https://github.com/WordPress/gutenberg/blob/9675b99c6e7c88af16749acc3dd0dcdfbe35db27/packages/block-editor/src/components/media-upload/README.md).

I'm ok if we land this in develop and see what that approach would look like in a follow-up branch, though.

Reply:
I like the idea, but maybe this is a bit out of scope of this PR. Maybe we could create an issue and we tackle it later?

Reply:
Related discussion should be tracked in #617 from now on, so marking this as resolved.